libbeat/beat/beat.go

```diff
@@ -447,6 +454,48 @@ func (b *Beat) loadDashboards() error {
 	return nil
 }
 
+func (b *Beat) loadTemplate() error {
+	if *setup {
```

Comment:
if you run `-setup` but have `setup.template.enabled: false` what is the expected behaviour?